Serve failed with showing local package paths

On `v1.6.3` after using `vendor/bin/rr get-binary` I've launched the `rr` binary file and got the next error output:
 ```
./rr serve -v -d
DEBU[0000] [http]: disabled
DEBU[0000] [metrics]: disabled
DEBU[0000] [headers]: disabled
DEBU[0000] [static]: disabled
DEBU[0000] [health]: disabled
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0x9365e0]

goroutine 1 [running]:
github.com/spiral/roadrunner/service/http.(*Service).AddMiddleware(...)
        /home/valery/Projects/opensource/roadrunner/service/http/service.go:66
github.com/spiral/roadrunner/service/gzip.(*Service).Init(0xc0000b8548, 0xc0000b7b24, 0x0, 0x0, 0x0, 0x0)
        /home/valery/Projects/opensource/roadrunner/service/gzip/service.go:23 +0x80
reflect.Value.call(0xc000192780, 0xc0000b8898, 0x13, 0xb0d681, 0x4, 0xc0000bbda0, 0x3, 0x4, 0xa06340, 0x0, ...)
        /usr/lib/go/src/reflect/value.go:460 +0x8ab
reflect.Value.Call(0xc000192780, 0xc0000b8898, 0x13, 0xc0000bbda0, 0x3, 0x4, 0x0, 0xc189c0, 0xc000192780)
        /usr/lib/go/src/reflect/value.go:321 +0xb4
github.com/spiral/roadrunner/service.(*container).initService(0xc0001080c0, 0xa5da20, 0xc0000b8548, 0x0, 0x0, 0x1, 0xbfb8c0, 0xc000097190)
        /home/valery/Projects/opensource/roadrunner/service/container.go:248 +0x1cc
github.com/spiral/roadrunner/service.(*container).Init(0xc0001080c0, 0xc01ee0, 0xc0000b86a0, 0x1, 0x1)
        /home/valery/Projects/opensource/roadrunner/service/container.go:149 +0xec
github.com/spiral/roadrunner/cmd/rr/cmd.init.0.func1()
        /home/valery/Projects/opensource/roadrunner/cmd/rr/cmd/root.go:109 +0x1f5
github.com/spf13/cobra.(*Command).preRun(0xc000138000)
        /home/valery/Projects/go_path/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:870 +0x49
github.com/spf13/cobra.(*Command).execute(0xc000138000, 0xc0000ad000, 0x2, 0x2, 0xc000138000, 0xc0000ad000)
        /home/valery/Projects/go_path/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:806 +0x138
github.com/spf13/cobra.(*Command).ExecuteC(0x1065920, 0xab0ec0, 0xc0001080d0, 0xb40528)
        /home/valery/Projects/go_path/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:945 +0x317
github.com/spf13/cobra.(*Command).Execute(...)
        /home/valery/Projects/go_path/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:885
github.com/spiral/roadrunner/cmd/rr/cmd.Execute()
        /home/valery/Projects/opensource/roadrunner/cmd/rr/cmd/root.go:69 +0x2d
main.main()
        /home/valery/Projects/opensource/roadrunner/cmd/rr/main.go:58 +0x364
```

--- a/build.sh
+++ b/build.sh
@@ -44,7 +44,7 @@ build_musl() {
   echo Packaging "$2" Build
   bdir=roadrunner-${RR_VERSION}-$1-$2-$3
   rm -rf builds/"$bdir" && mkdir -p builds/"$bdir"
-  CC=musl-gcc GOARCH=amd64 go build -ldflags "$LDFLAGS" -o "$OD/rr" cmd/rr/main.go
+  CC=musl-gcc GOARCH=amd64 go build -trimpath -ldflags "$LDFLAGS" -o "$OD/rr" cmd/rr/main.go
 
   mv rr builds/"$bdir"
 
@@ -68,4 +68,4 @@ if [ "$1" == "all" ]; then
   exit
 fi
 
-CGO_ENABLED=0 go build -ldflags "$LDFLAGS" -o "$OD/rr" cmd/rr/main.go
+CGO_ENABLED=0 go build -trimpath -ldflags "$LDFLAGS" -o "$OD/rr" cmd/rr/main.go

